improver_tests/acceptance/SHA256SUMS

```diff
@@ -462,6 +464,7 @@ b9d56c28ab78f296842865d3d6863d5017d6928e45632b00bf05f086fc651969  ./spot-extract
 b0cfd198d69d5d5f85775542c77dc4a2a0a3092d407e338d74c72618e5de7b1e  ./spot-extract/inputs/enukx_temperature_percentiles.nc
 7c1546a4a0eac805483d09a99268767ede51b5649eb545e002a1e02c9f3f43f2  ./spot-extract/inputs/enukx_temperature_realizations.nc
 f67aca830966488cb7ac76fbd91c50ccf2494898233557c3c4070244088a296c  ./spot-extract/inputs/enukx_temperature_thresholds.nc
+f84730d9815d1b488915eed704cadc311eb43135015395cb75cbd8cd4b9985cd  ./spot-extract/inputs/enukx_temperature_thresholds_multi_time.nc
```

Comment:
This isn't relevant to this PR

Reply:
Thanks. I've corrected this by branching from the 1.0.1 tag instead of master.